bundles/org.openhab.binding.worxlandroid/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>4.3.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.worxlandroid</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: Worx Landroid Binding</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>software.amazon.awssdk.iotdevicesdk</groupId>
+      <artifactId>aws-iot-device-sdk</artifactId>
+      <version>1.15.0</version>
+    </dependency>
+    <dependency>
+      <groupId>software.amazon.awssdk.crt</groupId>
+      <artifactId>aws-crt</artifactId>
+      <version>0.25.1</version>
+    </dependency>
+  </dependencies>
```

Comment:
This binding performs some similar requests/response, so it should be possible to create them without these SDK's.
Could you remove these dependencies?

Reply:
Can you be more explicit ? I'm afraid I did not get the meaning of your comment.
IIRW I tried to upgrade dependencies but it had major impact on code structure, so prefered to leave as it was.

Reply:
I meant to say that the SDK should not be used. Instead the httpclient can be used to perform the same api request/responses.

You could look at the salus binding that also holds a very light weight base class doing much of the SDK work without needing the actual SDK. 

Due to problems of the past with size, upgrades and other issues we try to keep these SDK’s out of the codebase.

Reply:
This is for Mqtt client of AWS IoT

Reply:
Have not looked in depth on how the MQTT client and websocket / authenticatino work together. I just know that we tent not to accept these SDK's. For the salus binding we had a similar issue with the `software.amazon.awssdk.crt` package. A fix was found by some lightweight classes that implemented the authentication. 
Hope you figure out hwo to get rid of the SDK with these suggestions, otherwise please post the specific problems you encounter and maybe we can find a fix together. If ultimately it seems not feasable, we can always merge it as is, but yeah really prefer not to..

Reply:
I tried to incorporate limite set of classes from the SDK but I think I fall in the same problem with the crt package. Will take a look on how it has been handled for Salus.

Reply:
@lsiepel : had no success here.

Reply:
We will need to look for a solution as the jar file seems to explode with this SDK. It grows to a whopping 17MB.

Reply:
Using `org.openhab.core.io.transport.mqtt` was my initial intention but as you identified it connection uses a mix of `mqtt` and `websocket` that I've not been able to reproduce in an other environment.

Reply:
Are you aware of a simulator or demo api? I don’t have such a device or account. To help figure out, this would be useful.

Reply:
Not as far as I know. I Can give you m'y credentials by PM if you agree

Reply:
@lsiepel : gentle ping.

Reply:
See also https://github.com/openhab/openhab-addons/pull/14669#issuecomment-1489217564.

Reply:
Have no0t forgotten this, but have not yet had the dedicated time to look at it. Would be nice if we can either build our own or use a lite mqtt over websockets client.